### ISSUE TITLE
Unable to access "banned ports"

### ISSUE THREAD

**Issue by sanjayankur31** [qutebrowser/qutebrowser#2925]
I was trying to access my znc bouncer's web admin which is on port 6666, and that is non standard. I get an `ERR_UNSAFE_PORT` and I haven't been able to find a way to work around this. In both FF and Chrome, these ports are "blocked" too, but there seem to be configuration options to get around this ban. 

Is such a config option available in qutebrowser? If not, can it please be added?

I'm using git snapshots. At the moment, I've got this:
qutebrowser-0.11.0-20170823git173688c.fc26.noarch

```
qutebrowser v0.11.0
Backend: QtWebEngine (Chromium 56.0.2924.122)

CPython: 3.6.2
Qt: 5.7.1
PyQt: 5.8.1

sip: 4.19.1
colorama: no
pypeg2: 2.15
jinja2: 2.9.6
pygments: 2.2.0
yaml: 3.12
cssutils: 1.0.1 $Id$
typing: yes
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: no
sqlite: 3.20.0
QtNetwork SSL: OpenSSL 1.0.2j-fips  26 Sep 2016

Style: Adwaita::Style
Platform: Linux-4.12.5-300.fc26.x86_64-x86_64-with-fedora-26-Twenty_Six, 64bit
Linux distribution: Fedora 26 (Workstation Edition) (fedora)
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /usr/lib64/qt5/libexec, data path: /usr/share/qt5

Paths:
config: /home/asinha/.config/qutebrowser
data: /home/asinha/.local/share/qutebrowser
system_data: /home/asinha/.local/share/qutebrowser
cache: /home/asinha/.cache/qutebrowser
download: /home/asinha/Downloads
runtime: /run/user/1000/qutebrowser
```

Cheers!

**Comment by pkillnine** [comment]
I also get `ERR_UNSAFE_PORT` when accessing `http://localhost:6666`.

**Comment by The-Compiler** [comment]
I'd [expect](https://superuser.com/questions/188006/how-to-fix-err-unsafe-port-error-on-chrome-when-browsing-to-unsafe-ports) it to work when adding `--qt-flag explicitly-allowed-ports=6666` to qutebrowser's invocation, but for some reason it still doesn't.

**Comment by The-Compiler** [comment]
I opened [QTBUG-62808](https://bugreports.qt.io/browse/QTBUG-62808) for this.

**Comment by The-Compiler** [comment]
Should be fixed in Qt 5.9.2 or 5.9.3 with https://codereview.qt-project.org/#/c/203810/

**Comment by sanjayankur31** [comment]
Brilliant! Thanks! :+1:
